packages/api/src/tasks/__tests__/checkAlerts.test.ts

```diff
@@ -404,14 +373,14 @@ describe('checkAlerts', () => {
         1,
         'https://hooks.slack.com/services/123',
         {
-          text: 'Alert for "Max Duration" in "My Dashboard" - 102 exceeds 10',
           blocks: [
             {
               text: {
                 text: [
                   `*<http://localhost:9090/dashboards/${dashboard._id}?from=1700170200000&granularity=5+minute&to=1700174700000 | Alert for "Max Duration" in "My Dashboard">*`,
                   'Group: "HyperDX"',
                   '102 exceeds 10',
+                  '',
```

Comment:
nit: should we just `trim()` the rendered string so we don't send extra new lines?

Reply:
good idea. let me do it in the next pr